personalwebsite/projects/templates/projects.html

```diff
@@ -10,28 +10,39 @@
 <div class="page-title border-bottom my-project">
     <h1 class="display-5 font-family-roboto">Projects</h1>
 </div>
-{% if all_projects %}
-{% for project in all_projects %}
-<div class="row">
-</div>
+{% if project_list %}
+{% for project in project_list %}
 <div class="row">
     <div class="col-xl-3 col-lg-4 col-md-12 col-sm-12 text-center my-3">
         {% if project.cover_image %}
         <a><img src="{{project.cover_image.url}}" height="256px" width="256px" style="border: 1px solid black;"></a>
         {% else %}
-        <a><img src="../media/project/cover_images/empty.jpg" height="256px" width="256px" style="border: 1px solid black; position: relative;"></a>
+        <a><img src="../media/project/cover_images/empty.jpg" height="256px" width="256px"
+                style="border: 1px solid black; position: relative;"></a>
         {% endif %}
     </div>
     <div class="col-xl col-lg-8">
         <div class="text-justify">
             <div class="proj-desc">
-                <h5>{{project.title}}</h5>    
+                <h5>{{project.title}}</h5>
```

Comment:
Consistency: Add a single space at the start and end inside the template tags {{ variable }}.
`{{project.title}}` -> `{{ project.title }}`

Reply:
Good point, I'll note this going forward.